core/src/main/resources/defaults/blueprints/hdf31-flow-management.bp

```diff
@@ -0,0 +1,92 @@
+{
+  "inputs": [],
+  "description": "Useful for data-flow management with Apache NiFi",
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdf31-flow-management",
+      "stack_name": "HDF",
+      "stack_version": "3.1"
+    },
+    "configurations": [
+      {
+        "nifi-ambari-config": {
+          "nifi.security.encrypt.configuration.password": "changemeplease",
+          "nifi.node.protocol.port": "9089",
+          "nifi.node.port": "9090",
+          "nifi.node.ssl.port": "9091",
+          "nifi.max_mem": "1g"
```

Comment:
I'm not sure if it is a good idea to override here. Ideally we should stay as close to the on-prem install as possible.
Is this tweak needed for on-prem too?

Experience from the NiFi-QE tests shows that the max_mem needs to be bumped to 4gigs and
timeouts needs to increased to pass the tests.

I'd suggest that we incorporate all those changes to Ambari's stack advisor.

Reply:
currently
the timeout configs mentioned by @tibkiss 
https://community.hortonworks.com/comments/142363/view.html

without them nifi-systemtests fail in about 5 minutes.